js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,14 @@
+function duplicateLetters(args) {
```

Comment:
`args` is a little ambiguous as it doesn't help the reader understand exactly what is expected in the method. `str` could have been a better name.